tests/ord-aggregator/tests/handler_test.go

```diff
@@ -459,13 +482,64 @@ func storeMappingBetweenORDAndInternalBundleID(t *testing.T, respBody string, nu
 	return ordAndInternalIDsMapping
 }
 
-func verifyEntitiesWithPublicVisibility(t *testing.T, httpClient *http.Client, publicEntitiesMap map[string]string, entity string, expectedNumberOfPublicEntities int) {
+func verifyEntitiesWithPublicVisibilityInORD(t *testing.T, httpClient *http.Client, publicEntitiesMap map[string]string, entity string, expectedNumberOfPublicEntities int) {
 	respBody := makeRequestWithHeaders(t, httpClient, testConfig.ORDServiceURL+fmt.Sprintf("/%s?$format=json", entity), map[string][]string{tenantHeader: {testConfig.DefaultTestTenant}})
 
 	assertions.AssertMultipleEntitiesFromORDService(t, respBody, publicEntitiesMap, expectedNumberOfPublicEntities, descriptionField)
 	t.Logf("Successfully verified public %s", entity)
 }
 
+func verifyEntitiesVisibilityViaGraphql(t *testing.T, clientWithInternalScope, clientWithoutInternalScope *gcli.Client, entitiesMap, publicEntitiesMap map[string]string, expectedNumberOfEntities, expectedNumberOfPublicEntities int, appID string, isAPIDef bool) {
```

Comment:
Can we invoke the method just once after events are verified and do both asserts?

This way we can get rid of the bool parameter and the if as well as make 2 fewer calls to director which will speed things up.